src/applications/gi/components/search/InstitutionSearchForm.jsx

```diff
@@ -96,7 +96,7 @@ function InstitutionSearchForm({
             </div>
           )}
 
-          <h2>{keywordLabel}</h2>
+          <h3>{keywordLabel}</h3>
```

Comment:
does this cause axe issues?
should this be done with `vads-u-font-size--h3` css class instead?

Reply:
Good call, it did fail Axe, I went ahead updated it.